assets/css/WorkflowEditor/Report/_Block.scss

```diff
@@ -1,17 +1,32 @@
 article.report .report-container section.block {
+  padding: 1rem 2rem;
   display: flex;
-  margin: 1rem 2rem;
+  flex-direction: column;
```

Comment:
Please nix `display: flex` and `flex-direction: column`. Or add a comment saying why that doesn't work.